src/hooks/useWearables.ts

```diff
@@ -0,0 +1,114 @@
+import { useQuery } from 'react-query';
+import {
+  SyncTypeSettings,
+  ToggleWearableResult,
+  WearableIntegrationFailureCode,
+  WearableIntegrationStatus,
+  WearablesSyncState,
+} from '../components/Wearables/WearableTypes';
+import { useActiveAccount } from './useActiveAccount';
+import { useHttpClient } from './useHttpClient';
+
+/**
+ *
+ * @returns UTC offset in ±HHMM format, e.g., -04:00 for New York City during daylight saving time
+ * or -05:00 during standard time. Equivalent to moment's utcOffset() function.
+ */
+const getUTCOffset = () => {
+  const offsetInMinutes = new Date().getTimezoneOffset();
+  const sign = offsetInMinutes > 0 ? '-' : '+';
+  const hours = Math.abs(Math.floor(offsetInMinutes / 60))
+    .toString()
+    .padStart(2, '0');
+  const minutes = Math.abs(offsetInMinutes % 60)
+    .toString()
+    .padStart(2, '0');
+  const offset = `${sign}${hours}${minutes}`;
+  return offset;
+};
+
+export const useWearableIntegrations = (appId: string) => {
+  const { httpClient } = useHttpClient();
+  const { accountHeaders } = useActiveAccount();
+  return useQuery('/v1/wearables', () =>
+    httpClient
+      .get<WearablesSyncState>('/v1/wearables', {
+        params: { appId },
+        headers: accountHeaders,
+      })
+      .then((res) => res.data),
```

Comment:
Can we use `enabled` and check the presence of `accountHeaders` as a defense mechanism?